github/release_notes/util.py

```diff
@@ -472,12 +426,14 @@ def extract_release_notes(
     return release_notes
 
 
-def release_note_block(
+def create_release_note_block(
     code_block: dict,
     reference_type: ReferenceType,
     user_login: str,
-    cn_current_repo: ComponentName,
-    reference_id: str=None
+    current_component: gci.componentmodel.Component,
+    source_component: gci.componentmodel.Component = None,
+    reference_id: str=None,
+    cache_dir: str=None,
```

Comment:
rather name it component_descriptor_cache_dir?
also, I tend to dislike that this needs to be passed all over the place